### PR TITLE
feat: go1.21 release

### PR DESCRIPTION
**Pull request by hendriknielaender**

### CODE REVIEW

**Comment by flyck** [comment]
image

**Comment by flyck** [comment]
Tried also different queries:
image

**Comment by flyck** [comment]
dalle+midjourney

File: data/blog/go-1-21.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+c := max(1, 2.0)  // c == 2.0
+
+t := max("", "foo", "bar") // t == "foo"
+```
+
+## Profile Guided Optimization (PGO)
+
+Go 1.21 enables AutoFDO by default, a feature that was introduced in Go 1.20. AutoFDO uses profile data collected from the execution of instrumented binaries to guide the optimization of the code, leading to performance improvements.
+
+---
```
Comment (flyck) [inline data/blog/go-1-21.md:105]: Nit: Add a "Conclusion" Section like in but 0.6 for the summary

**Review by flyck — APPROVED** [review]
Only few nits, awesome post 🙌🏻

File: data/blog/go-1-21.md (hunk 1 above)
Comment (hendriknielaender) [reply data/blog/go-1-21.md:110]: done :)

File: data/blog/go-1-21.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+title: "Go 1.21: Now with More Gopher Power and Less 'Go-tchas'!"
+description: "Explore innovative features, including enhanced map and slice operations, structured logging..."
+image: ~/assets/images/thumbnails/gopher.png
+imageCreditUrl: https://labs.openai.com/
+tags: [gptpost, update, release, go, golang, code]
+---
+
+# The Exciting New Features of Go 1.21: A Deep Dive
+
+As August 2023 rolls around, it's time to turn our attention to the release of Go 1.21, which promises to make Gophers' lives even more comfortable. This release is chock-full of enhancements and new features focusing on improving the developer experience, performance, and quality of life.
```
[the remark this answers is not in the document]
Comment (hendriknielaender) [reply data/blog/go-1-21.md]: re-written

**Pull request merged**